app/models/maintenance_stats/stats/github/release_stats.rb

```diff
@@ -8,11 +8,13 @@ def initialize(dataset)
         end
 
         def get_stats
-          last_week_releases = @results.data.repository.releases.nodes.select {|release| release.published_at > @now - 7}.count
-          last_month_releases = @results.data.repository.releases.nodes.select {|release| release.published_at > @now - 30}.count
-          last_two_month_releases = @results.data.repository.releases.nodes.select {|release| release.published_at > @now - 60}.count
-          last_year_releases = @results.data.repository.releases.nodes.select {|release| release.published_at > @now - 365}.count
-          {
+          return {} if @results.nil?
```

Comment:
This is a slight change. Before we were sending back `0` releases, but our original stance was to return nothing for a stat if we didn't have the data for it. I think this is more correct.